### PR TITLE
Switch to query builder and db trait. Implement datbase support for pg/sqlite

### PR DESCRIPTION
**Pull request by dmzmk**
- Bumps axum to 0.7
- Bumps number of libs via `cargo update`
- Replaces hard coded db with db trait
- Replaces own error enum with anyhow and newtype wrapper
- Updates error logging
- Drops unused tables, renames records into message chunks

### CODE REVIEW

File: myceliald/src/http_client.rs [hunk 1]
```diff
@@ -233,6 +236,11 @@ impl HttpClient {
         rx: &mut Receiver<HttpClientMessage>,
     ) -> Result<(), SectionError> {
         let mut interval = tokio::time::interval(Duration::from_secs(5));
+        // set first tick to 50ms from now, allowing main application to set sonnection details
```
Comment (wwkeyboard) [inline myceliald/src/http_client.rs:239]: "connection"

File: myceliald/src/http_client.rs [hunk 2]
```diff
@@ -233,6 +236,11 @@ impl HttpClient {
...
+        // before first loop iteration
+        if let Some(delay) = tokio::time::Instant::now().checked_add(Duration::from_millis(50)) {
```
Comment (wwkeyboard) [inline myceliald/src/http_client.rs:241]: Minor nit, but this should probably be in the spawn function instead, it better communicates that it's setup before entering the loop and this function only handles the loop itself.

File: pipe/section/section_impls/mysql_connector/src/destination.rs [hunk 3]
```diff
@@ -140,13 +140,14 @@ impl Mysql {
...
-        TimeUnit::Microsecond => NaiveDateTime::from_timestamp_micros(t),
+        TimeUnit::Second => DateTime::from_timestamp(t, 0),
+        TimeUnit::Millisecond => DateTime::from_timestamp_micros(t * 1000),
+        TimeUnit::Microsecond => DateTime::from_timestamp_micros(t),
         TimeUnit::Nanosecond => {
-            NaiveDateTime::from_timestamp_opt(t / 1_000_000_000, (t % 1_000_000_000) as _)
+            DateTime::from_timestamp(t / 1_000_000_000, (t % 1_000_000_000) as _)
         }
     }
+    .map(|d| d.naive_utc())
```
Comment (wwkeyboard) [inline pipe/section/section_impls/mysql_connector/src/destination.rs:150]: This seems like an extra step, why is it necessary?

Comment (dmzmk) [reply pipe/section/section_impls/mysql_connector/src/destination.rs:150]: chrono deprecated previous methods, linter yelling loudly in gh actions

File: console/src/utils/constants.ts [hunk 4]
```diff
@@ -23,11 +23,6 @@ export enum NodeType {
   target = 'target',
 }
 
-export const headers = {
-  'Content-Type': 'application/json',
```
[the remark this answers is not in the document]
Comment (dmzmk) [reply console/src/utils/constants.ts:27]: yeah, axios picks content type appropriately

File: myceliald/src/http_client.rs (hunk 2 above)
Comment (dmzmk) [reply myceliald/src/http_client.rs]: moved sleep to spawn, @wwkeyboard does it look better now?

File: console/src/utils/constants.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (wwkeyboard) [reply console/src/utils/constants.ts:27]: Awesome!

**Review by wwkeyboard — APPROVED** [review]

**Pull request merged**